loadtest/set_workers.sh

```
@@ -0,0 +1,94 @@
+fi
+
+$DOCKER cp "$TMP" "$CONTAINER:/app/process_monitor.sh"
+$DOCKER exec "$CONTAINER" chmod +x /app/process_monitor.sh
+
+echo "== docker restart $CONTAINER"
+$DOCKER restart "$CONTAINER" >/dev/null
+
+echo -n "== waiting for /health "
+for _ in $(seq 1 120); do
+    code=$($DOCKER exec "$CONTAINER" curl -s -o /dev/null -w '%{http_code}' \
+           http://localhost:${PIPESHUB_QUERY_PORT}/health 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)